modules/core/src/matrix_transform.cpp

```diff
@@ -1097,4 +1101,50 @@ void rotate(InputArray _src, OutputArray _dst, int rotateMode)
     }
 }
 
+void rotate(InputArray _src, OutputArray _dst, int rotateMode)
+{
+    int angle;
```

Comment:
Missing checks of input parameters constraints.

Reply:
Check present in initial version of cv::rotate() is now present in the updated patch.
Thanks